docs/HowToGuides/FirstPullRequest.md

```diff
@@ -14,19 +14,17 @@ contribution process.
 ## How do I pick something to work on?
 
 In case you don't have something specific you'd like to work on, such as
-implementing something for a Swift Evolution pitch, you could start off by
-working on a bug labeled `StarterBug` on [Swift repository 'Issues' tab][StarterBug]. 
-If the issue hasn't been assigned to someone, check the comments in case someone has
-already started working on it. If not, feel free to assign it to yourself and
-start working on it!
-
-[StarterBug]: https://github.com/apple/swift/issues?q=is%3Aissue+is%3Aopen+label%3AStarterBug
+implementing something for a [Swift evolution proposal](https://www.swift.org/swift-evolution),
+you could start off by working on a [good first issue][good-first-issues]. If
+the issue hasn't been assigned to someone, check the comments in case someone
+has already started working on it. If not, feel free to assign it to yourself
+and start working on it!
```

Comment:
Can somebody without non-contributor/committer status actually do that (assign issues to themselves)?

Reply:
No, they cannot; good catch! How about suggesting to add a comment for now? Once the Contributor Experience Workgroup has a page on the website, we can link out to that page and ask to tag us for that stuff.

Reply:
Sounds good to me!